Doc/library/random.rst

```diff
@@ -294,6 +294,10 @@ be found in any statistics text.
 
    Return the next random floating point number in the range ``0.0 <= X < 1.0``
 
+   .. warning::
+      Calling this function in the concurrent use can cause performance degradation.
+      Instead, Consider using :class:`Random` per thread.
+
```

Comment:
Let's move this up to the module level, perhaps between the security warning and the "see also". The advice is not limited this function.

I think maybe a `.. note::` is more appropriate. A red "warning" seems a bit severe to me.

For the text, something like:

```
The global random number generator and instances of :class:`Random` are thread-safe. However, 
in the free-threaded build, concurrent calls to the global generator or to the same instance of :class:`Random`
may encounter contention and poor performance. Consider using separate instances of :class:`Random`
per thread instead.
```

Reply:
I agree.  This replaces my comment.